elements/forum/components/ProfileAside/ProfileAside.tsx

```diff
@@ -1,45 +1,49 @@
 import React from 'react';
+import Link from 'next/link';
 import Image from 'next/image';
-import { ProfileAside, ProfileAsideHeading, ProfileCondensedInfo } from './ProfileAside.style';
+import { format } from 'date-fns';
+import {
+  ProfileAside,
+  ProfileAsideFact,
+  ProfileAsideHeading,
+  ProfileCondensedInfo,
+  ProfileImageWrapper,
+} from './ProfileAside.style';
+import IUser from '../../../../models/IUser';
 
 interface ProfileAsideProps {
-  author?: string;
+  author?: IUser;
   // authorRegistered?: Date;
```

Comment:
remove undescriptive comments, if you want to keep them for later do that to find it when searching:

```suggestion
  // TODO: add state => authorRegistered?: Date;
```